Meetings/2019-12-03.md

```diff
@@ -1,25 +1,44 @@
 # TSC Meeting 2019 December 3rd @12PM MST.
-- Zoom Link: 
+- Zoom Link: https://zoom.us/j/662376664
 
 ### Attending
-- ??
+- @AndrewAnnex
+- @michaelaye
+- @rbeyer
+- @victoronline
 
 ## Agenda / Notes
 - Action Items that were not completed from last time
-
-- Report on further discussions with Meagon/Sarah about HQ education (@jlaura)
+	- The only Action Item that was not completed has been carried over to this meeting's Action Items
 
 - ISIS application status (@jessemapel)	
+	- The ISIS TC is moving onwards, we're still working on getting a charter approved. After that I hope to start working on policy documents.
 
 - PlanetaryPy application status (@michaelaye)
+	- A draft application is being worked in their repo: https://github.com/planetarypy/TC/blob/master/PSO-Application.md
+	- They are unsure when it will be formally submitted, but things are moving along.
 
 - LPSC Submission (all)
+	- We want to submit an abstract that basically describes what the PSO TSC is, and what it does (basically an existance statement).
+	- Would be nice if the ISIS TC and PlanetaryPy TC would also submit abstracts about themselves
```

Comment:
I am planning on submitting an abstract to LPSC about the ISIS TC, I am not sure if I will be attending the conference proper though.